NOM_visualizations/R/NOM_R_notebook.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>I would change this to "NOM analysis records" and then describe that these are records for the analysis, which connect the processed data objects to the raw data objects (from the instrument).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
changed the language, hopefully it makes more sense now?